packages/core/src/domain/configuration/endpointBuilder.ts

```diff
@@ -99,6 +89,30 @@ export function createEndpointBuilder(
     return shouldUseIntakeV2(endpointType) ? `/api/v2/${INTAKE_TRACKS[endpointType]}` : `/v1/input/${clientToken}`
   }
 
+  function buildQueryParameters(endpointType: EndpointType, source?: string) {
+    const tags =
+      `sdk_version:${sdkVersion}` +
+      `${env ? `,env:${env}` : ''}` +
+      `${service ? `,service:${service}` : ''}` +
+      `${version ? `,version:${version}` : ''}`
+
+    let parameters = `ddsource=${source || 'browser'}&ddtags=${encodeURIComponent(tags)}`
+
+    if (proxyHost) {
+      const datadogHost = buildHost(endpointType)
+      parameters += `&ddhost=${datadogHost}`
+    }
```

Comment:
It seems like you are adding the `ddhost` parameter twice (once here and ance in `build()`). Maybe add a test to check the full URL instead of `toContain` / `toMatch`?